File: src/main/java/it/contrader/dto/GraphDTO.java
package it.contrader.dto;

import com.fasterxml.jackson.annotation.JsonIdentityInfo;
import com.fasterxml.jackson.annotation.ObjectIdGenerators;

import it.contrader.model.Graph.FontStyle;
import it.contrader.model.Graph.TipoGrafico;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonIdentityInfo(generator=ObjectIdGenerators.PropertyGenerator.class, property="id")
public class GraphDTO {
	
	private Long id;
	
	private Boolean titoloBool;
	
	private String titolo;
	
	private FontStyle fontStyle;
	
	private Integer fontSize;
	
	private TipoGrafico tipografico;
	
	private String posTitolo;
	
	private Boolean legenda;
	
	private String posLegenda;
	
	private Boolean pareto;
	
}

File: Presentazione/Presentazione/DTO/GraphDTO.cs
using Presentazione.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Presentazione.DTO
{
    public class GraphDTO
    {
        public long id { get; set; }

        public long utente { get; set; }

        public string tipoGrafico { get; set; }

        public bool titoloBool { get; set; }

        public string titolo { get; set; }

        public string fontStyle { get; set; }

        public string fontColor { get; set; }

        public int fontSize { get; set; }

        public string posTitolo { get; set; }

        public bool legenda { get; set; }

        public string posLegenda { get; set; }

        public bool pareto { get; set; }

        public bool mixed { get; set; }

        public DateTime created { get; set; }

        public DateTime modify { get; set; }
    }
}
